drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -229,9 +229,10 @@ def outgoing_referral_postings(**args)
     end
 
     def external_form_submissions(**args)
-      # Not optimized performance, but reduces duplication of project_match logic
-      instances = Hmis::Form::Instance.with_role(:EXTERNAL_FORM) # Don't filter by active; show all past submissions
-      identifiers = instances.for_project_through_entities(object).select(:definition_identifier)
+      # Find form instances for this project. Only include active instances. (Unlike other form types, we do not support
+      # viewing "legacy" form submissions from inactive instances, to simplify implementation.)
+      instances = Hmis::Form::Instance.with_role(:EXTERNAL_FORM).for_project(object).active
```

Comment:
Perhaps add comment here about why we don't use `for_project_through_entities` (similar to pick_list_option above)